story/ch1/onik_tips_06.txt

```diff
@@ -1,58 +1,58 @@
-`"Of course, I doubt that theory.`
-` Since he has no car, one would expect him to have a limited area to which he could have escaped to. `
-`But within the station there were rumors going around that he had accidentally slipped and drowned in the swamp when he went to throw away the body."`
-`To the locals, that swamp is believed to be bottomless.`
-` It's known as Onigafuchi, the Demon's Abyss, and it's said that the bottom of the swamp is connected to the hellish world of demons.`
-`The atrocious demon from hell that was XX.`
-` Could it be that he had returned to hell through the swamp...?`
-`The Zijin Dragon, bringer of luck and fortune!`
-`-Prosperous future!-`
-` -Ladies leaping into your arms!-`
-` -Business success!-`
-` -Promotions!-`
-` -Ambition!-`
-` -Protection from harm!-`
-`Not just with pachinko and horse races, but business and even love...`
-`Guaranteed or your money back!`
-`Zijin Dragon Bracelet DX (One piece)`
-`27,800 Yen`
+`Hinamizawa barajının üzerine kâbus çöktü!`
+`Linç ve cinayet/parçalanma!`
+`X ayının X gününde, XX vilayetinin Shishibone Şehrindeki Hinamizawa baraj şantiyesinde tüyler ürpertici bir cinayet/parçalanma olayı yaşandı.`
+`Bu vaka takımadaları şok etmiş olsa da, polis vaka hakkında herhangi bir ayrıntı vermiyor...`
+` Hinamizawa barajında tam olarak ne oldu...?`
+`"Muhtemelen ilk başta onu öldürmek istememişlerdir,`
+` Ancak kurban bir kürek sallayarak direnince, diğer failler de karşılık vererek silahlandı ve olay hızla cinayete dönüştü."`
+`Dedi, bahsi geçen Müfettiş A.`
+`Bu kanlı trajedi sona erdikten sonra, kimsenin canlı olduğunu sanmayacağı bir cesetle baş başa kaldılar...`
+`XX-san kaba davranışlarıyla şüphelilere her gün eziyet etmişti.`
+`Başta bunun bir intikam olması gerekiyordu...`
+`"Suçluların hepsi yaptıkları karşısında dehşete düşmüştü`
+` ve hatta biri polise teslim olmuştu."`
+`Cesedi saklamayı öneren ise bu grubun elebaşı XX idi.`
+`İlk başta isteksizce de olsa, kısa süre sonra yakalanmak istemediklerini düşünmeye başladılar.`
+`İnşaat alanında, cesedi altı kişiyle birlikte saklayabilecekleri çok sayıda yer vardı...`
+`Aslında cesedi saklayıp bölgeyi terk etme planları vardı...`
+`Ancak elebaşı, diğer beş kişinin vicdanlarının bu yükü kaldıramayacağından korkarak`
+` onu polise ihbar etmelerini engellemek için korkunç bir yöntem buldu...`
+`Cesedi aralarında paylaştırmak ve her birini bir parçayı saklamaktan sorumlu tutmak gibi iğrenç bir yönteme başvurdu.`
+`"XX basit bir adam öldürme olayını korkunç bir parçalamaya dönüştürmüş ve aralarında bir birlik duygusu yaratmak için her bir faili zorla olaya dahil etmişti."`
+`Her biri katılmıştı.`
+` Ama bu ne anlama geliyordu?`
+` A kişisi isteksizce konuştu:`
+`"XX her birine saklayacakları bir parça kesmelerini emretmişti.`
+` Başta tereddüt ettiler ama kimse reddetmedi."`
+`İşi yarıda bırakmak yoktu... bunun anlamı buydu.`

```

Comment:
in for a penny, in for a pound

British. Ised to say that a person should finish what he or she has started to do even though it may be difficult or expensive.